### PR TITLE
use

### PR DESCRIPTION
**Pull request by edewit**
fixes issue [RHMAP-7030](https://issues.jboss.org/browse/RHMAP-7030)

### CODE REVIEW

**Comment by secondsun** [comment]
@edewit Is building the app in studio sufficient to verify?

**Comment by edewit** [comment]
@secondsun good question, I think this breaks studio builds

File: package.json [hunk 1]
```diff
@@ -2,20 +2,27 @@
...
   "version": "0.0.0",
-  "dependencies": {},
+  "dependencies": {
+    "backbone": "^1.3.3",
+    "bootstrap": "^3.3.6",
+    "fh-js-sdk": "^2.14.4",
+    "jquery": "^3.0.0",
+    "underscore": "^1.8.3"
+  },
   "devDependencies": {
```
Comment (jcesarmobile) [inline package.json:12]: I think you need `"browserify": "^13.0.1"` and  `"browserify-shim": "^3.8.12"` too

File: Gruntfile.js [hunk 2]
```diff
@@ -78,6 +78,10 @@ module.exports = function (grunt) {
     // Empties folders to start fresh
     clean: {
       server: '.tmp'
+    },
+
+    browserify: {
+      'www/main.js': ['www/lib/*.js', 'www/js/*.js']
```
Comment (jcesarmobile) [inline Gruntfile.js]: I think `'www/js/*.js'` should be `'www/js/**/*.js'`

File: www/js/views/hello.js [hunk 3]
```diff
@@ -1,3 +1,8 @@
+var App = require('./init');
```
Comment (jcesarmobile) [inline www/js/views/hello.js]: I think it should be `var App = require('../helpers/init');`

**Comment by jcesarmobile** [comment]
Added a few comments

And bootstrap js was updated to 3.3.6, but the bootstrap.css is the 3.0.3 version

**Comment by edewit** [comment]
@jcesarmobile good find, did these changes a bit to fast

**Comment by jcesarmobile** [comment]
Can you add the .map files for bootstrap too?

**Comment by jcesarmobile** [comment]
LGTM

**Pull request merged**